graphql/errors/errors.go

```diff
@@ -1,4 +1,4 @@
-package graphql
+package errors
```

Comment:
I don't think we need a separate errors package. These functions are only used by graphql and should just share the same package name.